src/components/Table/cells/DefaultCell/DefaultCell.tsx

```diff
@@ -27,13 +29,20 @@ function DefaultCell({ cell }: CustomCellProps) {
     }
   }, [value])
 
-  return (
-    <Tooltip data={tooltip?.toString()}>
+  const cellContent =
+    isLink && getUrl ? (
+      <Link to={getUrl(row.original)} className='table-link'>
+        <div className='table-default-cell' ref={ref}>
+          {value}
+        </div>
+      </Link>
+    ) : (
       <div className='table-default-cell' ref={ref}>
         {value}
       </div>
-    </Tooltip>
-  )
+    )
+
+  return <Tooltip data={tooltip?.toString()}>{cellContent}</Tooltip>
```

Comment:
do we need a variable here?
```suggestion

  return <Tooltip data={tooltip?.toString()}>{
  isLink && getUrl ? (
      <Link to={getUrl(row.original)} className='table-link'>
        <div className='table-default-cell' ref={ref}>
          {value}
        </div>
      </Link>
    ) : (
      <div className='table-default-cell' ref={ref}>
        {value}
      </div>
      }
      </Tooltip>
```

Reply:
I would still vote for a variable, but remade it it a bit